utils/kafka-test-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/ConsumerRecordHelper.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.engine.kafka
+
+import io.circe.Decoder
+import pl.touk.nussknacker.engine.api.CirceUtil
+
+import java.nio.charset.StandardCharsets
+import scala.reflect.{ClassTag, classTag}
+
+object ConsumerRecordHelper {
+
+  def as[T: Decoder : ClassTag](data: Array[Byte]): T = {
```

Comment:
It is json related. I think that we should mention about somewhere in some name.